project/profiles/capacity4more/modules/c4m/restful/c4m_restful_quick_post/c4m_restful_quick_post.module

```diff
@@ -272,9 +272,12 @@ function c4m_restful_quick_post_form($group_id) {
     }
   }
 
+  $user_may_create_discussion = c4m_user_is_site_admin() ?
```

Comment:
not sure which resources you're referring to, but, by elaborating I meant to explain __when__ the user is not allowed.
The fact that the user is not allowed is trivial from the code.